mapper/pojo/src/main/java/org/hibernate/search/mapper/pojo/mapping/spi/PojoMappingDelegate.java

```diff
@@ -22,4 +21,26 @@
 	<T> PojoSearchTargetDelegate<T> createPojoSearchTarget(Collection<? extends Class<? extends T>> targetedTypes,
 			PojoSessionContextImplementor sessionContext);
 
+	/**
+	 * @param type a Java type
```

Comment:
Same question here about the javadoc format?

Reply:
Done